src/subscribables/subscribable.js

```diff
@@ -75,6 +79,20 @@ var ko_subscribable_fn = {
                 ko.dependencyDetection.end(); // End suppressing dependency detection
             }
         }
+        // Notify subscribers to the ALL_EVENT event, but not if that was the event that was fired (prevent double events)
+        if (this.hasSubscriptionsForEvent(allEvent) && event != allEvent) {
```

Comment:
So, do you think copy/paste is better for this case?
Can you pass event name in another way?
Do you think it's ok that "all" has different event name in subscribe callback? What's about consistency?
How does other libraries implement similar functionality?